compiler/damlc/tests/daml-test-files/Loop.daml

```diff
@@ -5,26 +5,45 @@
 
 module Loop where
 
+import Daml.Script
+
 loop : Int -> ()
 loop x = loop (x + 1) 
 
--- @ERROR range=12:1-12:25; Evaluation timed out after 5 seconds
-loop_in_scenario_machine = scenario do
-  alice <- getParty "p"
-  pure $ loop 1
-  
 template Helper
   with
     p : Party
   where
     signatory p
-    choice Go : ()
+    choice Loop : ()
       controller p
       do
         pure $ loop 1
 
--- @ERROR range=27:1-27:23; Evaluation timed out after 5 seconds
-loop_in_ledger_machine = scenario do 
+-- @ERROR range=24:1-24:16; Evaluation timed out after 1 seconds
+scenarioMachine = scenario do
+  pure $ loop 1
+
+-- @ERROR range=28:1-28:32; Evaluation timed out after 1 seconds
+ledgerMachineFromScenarioSubmit = scenario do
    alice <- getParty "p"
-   alice `submit` createAndExercise (Helper alice) Go
-  
+   alice `submit` createAndExercise (Helper alice) Loop
+
+-- @ERROR range=33:1-33:40; Evaluation timed out after 1 seconds
+ledgerMachineFromScenarioSubmitMustFail = scenario do
+   alice <- getParty "p"
+   alice `submitMustFail` createAndExercise (Helper alice) Loop
```

Comment:
adding 2 test for `submitMustFail` to be sure the Cancellation is not interpreter as a command submission failure.